primitives/src/v0.rs

```diff
@@ -923,6 +923,74 @@ pub mod fisherman {
 	}
 }
 
+// TODO: add feature gate.
+/// Runtime metric operations.
+#[derive(Debug, PartialEq, Eq, Clone, Encode, Decode, TypeInfo)]
```

Comment:
I don't think all of these are needed.

Reply:
`v0` is deprecated and will be removed soon™

Reply:
Thanks. Got it.